compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/ConfigUtils.cpp

```diff
@@ -199,6 +199,8 @@ LogicalResult setMatmulLoweringConfig(IREE::GPU::TargetAttr target,
   attrs.emplace_back(StringAttr::get(context, "subgroup"),
                      b.getI64ArrayAttr(subgroupTileSizes));
   attrs.emplace_back(StringAttr::get(context, "mma_kind"), mmaKind);
+  attrs.emplace_back(StringAttr::get(context, "promote_operands"),
+                     b.getI64ArrayAttr({0, 1}));
```

Comment:
There are quite a few hard coded strings here now. Maybe you can expose `getTilingLevelName`, and use it for the tiling levels, and something similar for `"mma_kind"` and `"promote_operands"`. Or add some setters for the different attribute fields and use those.